lib/npm-check-updates.js

```diff
@@ -346,6 +346,10 @@ function findPackage() {
     var pkgFileName = getPackageFileName();
 
     function getPackageDataFromFile(pkgFile, pkgFileName) {
+        // exit if no pkgFile to read from fs
+        if (pkgFile === null) {
+            programError(chalk.red(pkgFileName + ' not found'));
```

Comment:
Thanks! Could you have it print the same error that it did before?
https://github.com/tjunnone/npm-check-updates/pull/325/files#diff-1415f5139deb67a37325f1bde416bcbdL416